stdlib/2/os/__init__.pyi

```diff
@@ -219,9 +219,9 @@ def renames(old: _PathType, new: _PathType) -> None: ...
 def rmdir(path: _PathType) -> None: ...
 def stat(path: _PathType) -> Any: ...
 @overload
-def stat_float_times(newvalue: bool = ...) -> None: ...
-@overload
 def stat_float_times() -> bool: ...
+@overload
+def stat_float_times(newvalue: bool = ...) -> None: ...
```

Comment:
Shouldn't we just remove the `= ...` here?

Reply:
Done.